src/components/RadioWithTextField.tsx

```diff
@@ -118,19 +160,21 @@ export function RadioWithTextField({
               helperText={
                 errorText !== ''
                   ? errorText
-                  : field === 'isFromIPA'
-                  ? ''
-                  : t('additionalDataPage.allowedCharacters')
+                  : fieldIsFromIPA
+                    ? ''
+                    : t('additionalDataPage.allowedCharacters')
               }
               fullWidth
               sx={{
                 color: theme.palette.text.secondary,
-                marginBottom: field === 'isFromIPA' ? 3 : 0,
+                marginBottom: fieldIsFromIPA ? 3 : 0,
               }}
-              inputProps={{ maxLength: field === 'isFromIPA' ? 16 : 300 }}
+              inputProps={{ maxLength: fieldIsFromIPA ? 16 : 300 }}
               onChange={(e: any) => {
                 onTextFieldChange(true, field, e.target.value);
               }}
+              value={fieldIsFromIPA && checked === true && isIPA? ipaCode : undefined}
```

Comment:
Remove strict equal true, the variable alone is enough

Reply:
right